### ISSUE TITLE
[Suggestion] Total Sum of Passives per Line / Teamwide?

### ISSUE THREAD

**Issue by Seven7hNight** [zklm/ssherder-issues#22]
Not sure if it's possible, but I was wondering if, on the Team Formation, there could be a table which shows the calculation of the total effects of passives on a position / team wide?

For example, if Lynia and and Bell are on the line, their total effect on the position is:
Attack Power: 30% + 30%
Action Speed:  +25%
Reflex: 25% + 30%
Crit Rate: +30%
(Is it additive? I forgot how it's calculated, so assuming it's additive, the total instead of each source would work.)

Possibly also a (Potential) total for any stacking abilities
So if V. Beth was on the line, it would look something like that

Attack Power: 30% + 30%
Action Speed:  +25%
Reflex: 25% + 30% (+20%)[x3]
Crit Rate: +30% (+20%)[x3]
(Also, if it's additives, perhaps just the Total + [Max Potential w/ stacks] would suffice)?

Another table which shows the total amount of debuffs against the position.

I know I used midline as an example, but I feel like it could be useful when trying to calculate passives in the backline for those of us who are trying to balance between totems that are mostly for HP/Attack/CAR (Yuri) and totems that are mostly for with defensive stats like Defense/CritResist/Damage Reduction (Kevin).

Not sure if this would be absurdly complicated, but just a little something I'd like to suggest for the the stat calc autist in some of us.

**Comment by zklm** [comment]
Definitely, I really want to expand the formations section to include lots of information about stat gains from skill effects, including the designated ace, totems, etcetera

I've been taking a bit of a break from development to get back in to health, I think I should be able to get this done in July/August.
